engine.py

```diff
@@ -154,6 +154,10 @@ def _load_otls():
                 if self._houdini_version > (12, 5, 0):
                     menu_file = menu_file + ".xml"
 
+                self._menu_name = "Flow Production Tracking"
```

Comment:
I think that `self._menu_name = "Flow Production Tracking"` doesn't depend on the previous condition `if commands and enable_sg_menu:` so maybe it is possible to set it as the default menu name in `init_engine(self)` or maybe at the beginning of class:
```
class HoudiniEngine(sgtk.platform.Engine):
    """
    Houdini Engine implementation
    """
    
    ...
    
    _menu_name = "Flow Production Tracking"
```

Reply:
Good point. Not in `init_engine` since it's deprecated (https://github.com/shotgunsoftware/tk-core/blob/v0.21.6/python/tank/platform/engine.py#L172) but I can move it to `pre_app_init`